src/content/documentation/develop/web-ext-command-reference.md

```diff
@@ -340,6 +340,14 @@ web-ext run --start-url www.mozilla.com --start-url developer.mozilla.org
 
 Environment variable: `$WEB_EXT_START_URL`
 
+#### `--watch-file`
```

Comment:
Nit, `--watch-file`, `--watch-files` (because we added the alias in web-ext v.6.0.0, mozilla/web-ext#2182)